src/guillotine/fetchContentPathsForLocale.ts

```diff
@@ -1,9 +1,4 @@
-import type {
-    ContentApiBaseBody,
-    ContentPathItem,
-    GuillotineResult,
-    ProjectLocaleConfig,
-} from '../types';
+import type {ContentApiBaseBody, ContentPathItem, GuillotineResult, ProjectLocaleConfig} from '../types';
```

Comment:
Your code editor seems to break column 80 all the time?

Reply:
We have 140 chars length set in enonic codestyle, but there is chopping down if it exceeds it